.github/workflows/clang-tidy.yml

```diff
@@ -9,9 +9,12 @@ jobs:
     name: clang-tidy
     runs-on: ubuntu-latest
     steps:
+      - name: Install Boost
+        run: sudo apt install -y libboost-all-dev
+
       - name: Install clang-tidy
         run: |
-          sudo apt-get install -y clang-tidy-9
+          sudo apt install -y clang-tidy-9
```

Comment:
At least last I checked, `apt-get` (and `apt-cache`) was (were) recommended for CI because `apt` didn't have a committed non-interactive mode, and it will spit out a warning when run in such conditions.